xml/hwbp_registers.xml

```diff
@@ -1166,12 +1302,17 @@
             This field should be tied to 0 when S-mode is not supported.
         </field>
         <field name="sselect" bits="1:0" access="WARL" reset="0">
-            0: Ignore \FcsrTextraThirtytwoSvalue.
+            <value v="0" name="ignore">
+            Ignore \FcsrTextraThirtytwoSvalue.
+            </value>
 
-            1: This trigger will only match if the low bits of
+            <value v="1" name="equal">
```

Comment:
I would say that the names are 1=scontext and 2=asid.

Reply:
Done.